dao/src/main/java/greencity/repository/EventRepo.java

```diff
@@ -97,4 +98,20 @@ public interface EventRepo extends JpaRepository<Event, Long>, JpaSpecificationE
      */
     @Query(value = "SELECT e FROM Event e LEFT JOIN e.followers AS f WHERE f.id = :userId")
     Page<Event> findAllFavoritesByUser(Long userId, Pageable pageable);
+
+    /**
+     * Get all user's favorite events by user id.
+     *
+     * @param userId {@link Long}.
+     */
+    @Query(value = "SELECT e FROM Event e LEFT JOIN e.followers AS f WHERE f.id = :userId")
+    Set<Event> findAllFavoritesByUser(Long userId);
```

Comment:
Should it be Pagable?
Users could follow many events

Reply:
We already have `Page<Event> findAllFavoritesByUser(Long userId, Pageable pageable)`
Why do we need this one?